src/System.Security.Permissions/ref/System.Security.Permissions.csproj

```diff
@@ -42,7 +42,21 @@
     <ProjectReference Include="..\..\System.Text.RegularExpressions\ref\System.Text.RegularExpressions.csproj" />
     <ProjectReference Include="..\..\System.Threading\ref\System.Threading.csproj" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp3.0'">
+    <Reference Include="netstandard" />
```

Comment:
This was a new reference that I had to make the build work for this file. let me know if we need to change it